classes/jazzquiz_session.php

```diff
@@ -139,19 +139,33 @@ public function user_name_for_answer($userid) {
         if (!$user) {
             return '?';
         }
-        return fullname($user);
+        $userFirstLastName = $user->lastname . ', ' . $user->firstname;
+        return $userFirstLastName;
```

Comment:
Did you intend to change the result of `user_name_for_answer()`? The two methods are identical now.

Reply:
I've reversed it so that the surname is first instead of the first name. Is that okay or would you rather have it fullname($user)? I think university staff finds it more important to be able to sort by surname rather than name.